courses/machine_learning/time_series_prediction/modeling_bqml.ipynb

```diff
@@ -0,0 +1,1162 @@
+{
```

Comment:
<p>ORDER BY iteration</p><p> </p><p>below.. I would add in a sub-section. Compute the accuracy of the model on the eval set, comparing to a simple baseline model that just guesses the majority class. E.g., is this model doing better than a naive strategy?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/624/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Yeah. Good idea. I added something after to compute the accuracy on the eval data using just the majority guess. It's just less than BQML, phew!</p>